### PR TITLE
Refactor: Migrate `WalletFilterSidebar` to Chakra

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description

Migrates `WalletFilterSidebar` component to use Chakra UI.

## Related Issue
#8632

### CODE REVIEW

File: src/components/FindWallet/WalletFilterSidebar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  return (
+    <Accordion
+      as={VStack}
+      allowMultiple
+      reduceMotion
+      spacing={4}
+      alignItems="normal"
+      p={{ base: 4, sm: 0 }}
+      // Workaround to not having a dedicated prop to all items open by default
+      defaultIndex={Object.keys(filterOptions).map((key) => +key)}
```
Comment (TylerAPfledderer) [inline src/components/FindWallet/WalletFilterSidebar/index.tsx:69]: This keeps inline with the current behavior.

File: src/components/FindWallet/WalletFilterSidebar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+            key={uniqueId("walletFilterSidebarItem")}
+            background={filterPanelBg}
+            borderRadius="base"
+            // Remove border color from global style
+            borderColor="transparent"
+            p={6}
+          >
+            {({ isExpanded }) => (
+              <>
+                <chakra.h3
```
Comment (pettinarip) [inline src/components/FindWallet/WalletFilterSidebar/index.tsx:83]: Curious about the difference in doing `<Text as="h3">` here.

File: src/components/FindWallet/WalletFilterSidebar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+                        <SimpleGrid
+                          key={uniqueId("walletFilterSidebarItemPanel")}
+                          templateColumns="28px auto 34px"
+                          alignItems="center"
+                          columnGap={2.5}
+                          cursor="pointer"
+                          onClick={
+                            item.filterKey
+                              ? () => {
+                                  console.log(item.filterKey)
```
Comment (pettinarip) [inline src/components/FindWallet/WalletFilterSidebar/index.tsx:124]: Not sure if these were before but lets remove the consoles

File: src/components/FindWallet/WalletFilterSidebar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+                                  trackCustomEvent({
+                                    eventCategory: "WalletFilterSidebar",
+                                    eventAction: `${filterOption.title}`,
+                                    eventName: `${item.filterKey} ${!restProps
+                                      .filters[item.filterKey!]}`,
+                                  })
+                                  updateFilterOption(item.filterKey)
+                                }
+                              : () => {
+                                  console.log(item.filterKey)
```
Comment (pettinarip) [inline src/components/FindWallet/WalletFilterSidebar/index.tsx:134]: ✂️

File: src/components/FindWallet/WalletFilterSidebar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+                                conditionItem={item.showOptions}
+                              />
+                            )}
+                          </GridItem>
+                          <GridItem
+                            as="span"
+                            color="text200"
+                            fontSize="0.9rem"
+                            lineHeight="1.1rem"
+                            colStart={2}
```
Comment (pettinarip) [inline src/components/FindWallet/WalletFilterSidebar/index.tsx:187]: Nice! liked the structure!

**Review by pettinarip — CHANGES_REQUESTED** [review]
Loved this! 💯

Great usage of the Chakra components @TylerAPfledderer.

Left a couple of small questions and a request on the hook's name.

File: src/components/FindWallet/WalletFilterSidebar/index.tsx (hunk 2 above)
Comment (TylerAPfledderer) [reply src/components/FindWallet/WalletFilterSidebar/index.tsx:83]: Nope, this was in error! 😄

File: src/components/FindWallet/WalletFilterSidebar/index.tsx (hunk 3 above)
Comment (TylerAPfledderer) [reply src/components/FindWallet/WalletFilterSidebar/index.tsx:124]: Whoops! That was me.

File: src/components/FindWallet/WalletFilterSidebar/index.tsx (hunk 4 above)
Comment (TylerAPfledderer) [reply src/components/FindWallet/WalletFilterSidebar/index.tsx:134]: Man I'm doing terrible here 😆

**Comment by TylerAPfledderer** [comment]
@pettinarip should be good to go!

**Pull request merged**